src/main/java/racing/ui/InputView.java

```diff
@@ -1,19 +1,22 @@
 package racing.ui;
 
-import racing.domain.CarValidator;
+import racing.domain.CarNameValidator;
 import racing.exception.CarNameLengthException;
 
 import java.util.Scanner;
 
 public class InputView {
     private static final Scanner scanner = new Scanner(System.in);
```

Comment:
`static final` 키워드를 통해 상수로 선언하셨으니, 관례에 따라 이름을 대문자로 지어보면 어떨까요?
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```